Automattic/wp-calypso
# ETK: Fix PHPCS issues

#### Changes proposed in this Pull Request
Fixes all phpcs issues in the Editing Toolkit Plugin.

cc @kraftbj, I put this together today because I ran into some linter issues on wpcom, so I needed to make sure they were all cleaned up here before moving forward with some other work I'm doing. This only applies to the editing toolkit; some of the other calypso apps might still need to be fixed.

There were a few directories which I've ignored for these reasons:

- `*.asset.php`: These are autogenerated by webpack and is considered a compiled asset.
- `editing-toolkit/dev-plugin`: I'm pretty sure we can remove this soon, but it is not deployed. It is mostly for testing, so I don't want to manually fix the issues in it if we're going to remove it soon.
- I've ignored one specific rule in the synced newspack blocks. This rule is not covered by their phpcs config, but it is by ours. It's only a style issue, so it's not important to be strict about.
-
#### Testing instructions
1. Checkout this PR.
2. Run `composer install` and `yarn install` from wp-calypso root.
3. Run `yarn build` from `apps/editing-toolkit`
4. Run `yarn lint:php`. There should be no problems.
5. Try running `../../vendor/bin/phpcs` from that directory as well. There may be errors in other calypso apps, but none should be in the editing-toolkit directory, other than the ones above
6. All checks should pass on this PR.  (e.g. phpunit and phpcs)
7. The plugin should continue working correctly.

Related to #49579
